airbyte-integrations/connectors/source-freshdesk/source_freshdesk/spec.json

```diff
@@ -22,6 +22,12 @@
         "title": "Requests per minute",
         "type": "integer",
         "description": "Number of requests per minute that this source allowed to use."
+      },
```

Comment:
why don't we instead introduce a parameter for 30 days? make default = 30 days, and allow to override it?
lets say `lookback_days` ?